packages/service-library/src/servicelib/aiohttp/application.py

```diff
@@ -1,60 +1,82 @@
 import asyncio
 import logging
-from typing import Dict, Optional
 
 from aiohttp import web
 
 from .application_keys import APP_CONFIG_KEY, APP_FIRE_AND_FORGET_TASKS_KEY
 from .client_session import persistent_client_session
 
-logger = logging.getLogger(__name__)
+_logger = logging.getLogger(__name__)
 
 
-async def startup_info(app: web.Application):
-    print(f"INFO: STARTING UP {app}...", flush=True)
+async def _first_call_on_startup(app: web.Application):
+    # NOTE: name used in tests to mocker.spy
+    _logger.info("Starting %s ...", f"{app}")
 
 
-async def shutdown_info(app: web.Application):
-    print(f"INFO: SHUTTING DOWN {app} ...", flush=True)
+async def _first_call_on_cleanup(app: web.Application):
+    # NOTE: name used in tests to mocker.spy
+    _logger.info("Shutdown completed. Cleaning up %s ...", f"{app}")
 
 
-async def stop_background_tasks(app: web.Application):
-    running_tasks = app[APP_FIRE_AND_FORGET_TASKS_KEY]
-    task: asyncio.Task
-    for task in running_tasks:
+_MAX_WAIT_TIME_TO_CANCEL_SECONDS = 5
```

Comment:
We are using this naming convention elsewere 
```suggestion
_MAX_WAIT_TIME_TO_CANCEL_S: Final[int] = 5
```

Reply:
I think `SECONDS` is more clear than `S`. Moreover is a protected variable  :-)